client/python/cryoet_data_portal/src/cryoet_data_portal/_codegen.py

```diff
@@ -306,7 +315,7 @@ def _parse_model_list_field(
     of_model = GQL_TO_MODEL_TYPE.get(of_type.name)
     if of_model is not None:
         source_model = GQL_TO_MODEL_TYPE[gql_type.name]
-        source_field = to_snake(source_model)
+        source_field = to_snake(_get_remote_field_name_by_type(of_type, gql_type))
```

Comment:
Returning `None` from `_get_remote_field_name_by_type` seems unexpected. At a minimum, I'd expect to log an error and return `None`. Alternatively, consider just raising either here or in the function. Continuing does not seem like an option because `ListRelationship(., ., None)` should break at some point right?

Reply:
Ignore this, I did not read the following correctly at all!

Reply:
I clearly still need to wake up (I was looking at `main` for my second comment), but I think my first comment does stand.

Reply:
No problem - I've added a patch. mind taking a look?